components/filter/filter.js

```diff
@@ -227,7 +227,18 @@ class Filter extends LocalizeCoreElement(RtlMixin(LitElement)) {
 	}
 
 	_buildHeader(singleDimension) {
-		if (!this._activeDimensionKey && !singleDimension) return null;
+		if (!this._activeDimensionKey && !singleDimension) {
+			return html`
+				<div slot="header">
```

Comment:
Is the `<div>` needed, or could the `slot="header"` go right on the subtle button?

Reply:
I had it so this style would get applied: https://github.com/BrightspaceUI/core/blob/main/components/filter/filter.js#L53-L55
But looks like if I remove the `div` there too, everything works properly

Reply:
Ah ok cool. Yeah fewer elements is always better!